unity-environment/Assets/ML-Agents/Scripts/Agent.cs

```diff
@@ -119,7 +119,7 @@ public abstract class Agent : MonoBehaviour
     private int id;
 
     /// <summary>
-    /// Unity method called when the agent is istanciated or set to active.
+	/// Unity method called when the agent is instantiated or set to active.
```

Comment:
ensure that tabs are saved as spaces

Reply:
I've fixed that and all the other places in the code base.